docs/docs/command-line-interface.mdx

```diff
@@ -272,7 +298,7 @@ should be published.
 ```text [rasa export --help]
 ```
 
-## Start Rasa X
+## rasa x
 
 Rasa X is a toolset that helps you leverage conversations to improve your assistant.
 You can find more information about it <a className="reference external" href="https://rasa.com/docs/rasa-x/" target="_blank">here</a>.You can start Rasa X locally by executing
```

Comment:
looks like the :::note after this did not make it through the mirgration, would you mind fixing that?